.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+## Description
+
+Please write a few sentences describing the overall goals of the pull request's commits.
+
+
+
+## PR checklist
+
+Please remove the segment/s on either side of the | symbol as appropriate, and add any relevant link/s to the end of the line.
+If the provided content is part of the present PR remove the # symbol.
+
+- [ ] **changelog** provided | not required because: 
+- [ ] **development PR** provided # | not required because: 
```

Comment:
Should we not have a development/3.6 PR field for pull requests that must target MbedTLS and current TF-PSA-Crypto?

Reply:
Good point, in the middle/long term, it should be rare that a TF-PSA-Crypto PR breaks mbedtls and there is a need for an mbedtls companion PR, and thus there is not really a need for an mbedtls development PR line here. But in the short term it is more likely that changes in TF-PSA-Crypto break mbedtls:development. Thus adding the line. Otherwise there is no relationship between TF-PSA-Crypto and mbedtls-3.6 but through the framework. But in that case we will rather do the link through the framework PR I'd say.